packages/flutter_tools/test/general.shard/android/android_device_test.dart

```diff
@@ -163,6 +163,65 @@ void main() {
         FileSystem: () => MemoryFileSystem(),
         ProcessManager: () => mockProcessManager,
       });
+
+      final Map<String, TargetPlatform> targetPlatformMap = <String, TargetPlatform>{
```

Comment:
I don't see a place where we verify that the correct target platform is used

Reply:
> I don't see a place where we verify that the correct target platform is used

First, I find android_device_test is not running on a real device.
Second, with help of 'target-platform'  , we can build armv7 apk, when connect with an arm64 phone.
Do you have any good suggestions ？

Reply:
The android arch is passed to a Gradle build here:

https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/android/android_device.dart#L517

you'd need some verification that the override value is passed instead